dandiapi/api/tasks/scheduled.py

```diff
@@ -63,6 +64,24 @@ def aggregate_assets_summary_task(version_id: int):
     version_aggregate_assets_summary(version)
 
 
+@shared_task(soft_time_limit=30)
+def recalculate_missing_sha256_checksums():
+    # Only look at blobs that are over a day old
+    date_threshold = datetime.now(tz=UTC) - timedelta(days=1)
+    blobs_missing = AssetBlob.objects.filter(
+        sha256__isnull=True, created__lte=date_threshold
+    ).values_list('blob_id', flat=True)
+
+    num_missing = blobs_missing.count()
+    if num_missing == 0:
+        logger.info('No blobs with missing checksums found')
+        return
+
+    logger.info('Found %s blobs with missing checksums', num_missing)
+    for blob_id in blobs_missing:
```

Comment:
```suggestion
    for blob_id in blobs_missing.iterator():
```

It most likely won't ever happen that this queryset is so large that it becomes a problem, but since we're inside a celery task it costs nothing to use `.iterator()` here so :shrug: